core/src/main/java/com/griefcraft/listeners/LWCEntityListener.java

```diff
@@ -124,4 +141,141 @@ public void onEntityExplodeMonitor(EntityExplodeEvent event) {
+		if (!LWC.ENABLED || event.isCancelled()) {
+			return;
+		}
+		final ItemStack inHand = event.getItem();
+		if (inHand != null && 
+				(inHand.getType() == Material.ARMOR_STAND
+				|| inHand.getType() == Material.MINECART
+				|| inHand.getType() == Material.HOPPER_MINECART
+				|| inHand.getType() == Material.STORAGE_MINECART)) {
+			// actual location of an armor stand is x+0.5 z+0.5 from the air block coords
+			// minecarts spawn at y+0.0625, though
+			final Location l = event.getClickedBlock().getRelative(inHand.getType() == Material.ARMOR_STAND ? event.getBlockFace() : BlockFace.SELF).getLocation();
+			playerCreatedEntities.put(event.getPlayer(), l);
+		}
+	}
+	
+	@EventHandler
+	public void onMinecartCreate(VehicleCreateEvent event) {
+		final Entity entity = event.getVehicle();
+		final Location l = entity.getLocation();
+		// try to find this in the playerCreatedEntities map
+		for (Map.Entry<Player, Location> a : playerCreatedEntities.entrySet()) {
+			if (a.getValue().distanceSquared(l) < 1) {
+				entityCreatedByPlayer(entity, a.getKey());
+				playerCreatedEntities.remove(a.getKey());
+				break;
+			}
+		}
+	}
+
+	@EventHandler(ignoreCancelled = true, priority = EventPriority.MONITOR)
+	public void onCreateSpawn(CreatureSpawnEvent event) {
+		if (!LWC.ENABLED || event.isCancelled()) {
+			return;
+		}
+		final Entity entity = event.getEntity();
+		
+		final Location l = entity.getLocation();
+		
+		// try to find this in the playerCreatedEntities map
+		for (Map.Entry<Player, Location> a : playerCreatedEntities.entrySet()) {
+			if (a.getValue().distanceSquared(l) < 1) {
+				entityCreatedByPlayer(entity, a.getKey());
+				playerCreatedEntities.remove(a.getKey());
+				break;
+			}
+		}
+	}
+
+	private void entityCreatedByPlayer(Entity entity, Player player) {
+		LWC lwc = plugin.getLWC();
+
+		Protection current = lwc.findProtection(entity.getLocation());
+		if (current != null) {
+			if (!current.isBlockInWorld()) {
+				lwc.log("Removing corrupted protection: " + current);
+				current.remove();
+			} else {
+				if (current.getProtectionFinder() != null) {
+					current.getProtectionFinder().fullMatchBlocks();
+					lwc.getProtectionCache().addProtection(current);
+				}
+				return;
+			}
+		}
+		
+		if (!lwc.isProtectable(entity.getType())) {
+			return;
+		}
+
+		String autoRegisterType = lwc.resolveProtectionConfiguration(entity.getType(), "autoRegister");
+
+		if ((!autoRegisterType.equalsIgnoreCase("private"))
+				&& (!autoRegisterType.equalsIgnoreCase("public"))) {
+			return;
+		}
+
+		if (!lwc.hasPermission(player, "lwc.create." + autoRegisterType, new String[]{"lwc.create", "lwc.protect"})) {
+			return;
+		}
+		
+		Protection.Type type = null;
+		try {
+			type = Protection.Type.valueOf(autoRegisterType.toUpperCase());
+		} catch (IllegalArgumentException e) {
+			return;
+		}
+		if (type == null) {
+			player.sendMessage("§4LWC_INVALID_CONFIG_autoRegister");
```

Comment:
Missing translation?

Reply:
This is actually copied from onBlockPlaceMonitor()

Reply:
Oh, interesting. Probably ought to be changed there as well.